src/styles/blocks/card.scss

```diff
@@ -0,0 +1,94 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: $card-width;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  &__product-image {
+    @include product-image-size(
+      $card-product-img-width,
+      $card-product-img-height
+    );
+    margin-bottom: 40px;
+  }
+
+  &__product-title {
+    font-weight: 500;
+    font-size: $medium-font-size;
+    line-height: 18px;
+    color: $color-dark;
+
+    margin-bottom: 4px;
+  }
+
+  &__product-code {
+    font-weight: 400;
+    font-size: $small-font-size;
+    line-height: 14px;
+    color: $color-grey;
+
+    align-self: start;
+    margin-bottom: 16px;
+  }
+
+  &__elements-wrapper {
+    @extend %flex-space-between-center;
+    width: $card-elements-wrapper-width;
+
+    &--reviews {
+      margin-bottom: 24px;
+    }
+
+    &--price {
+      margin-bottom: 16px;
+    }
```

Comment:
I don't like these modifiers, better create new elements `__reviews` and `__price` and use your `%flex-space-between-center` placeholder there